libs/platform/src/lib/form/radio-group/radio-group.component.ts

```diff
@@ -225,32 +211,39 @@ export class RadioGroupComponent
             return;
         }
 
-        let firstEnabledButtonIndex = -1;
         this._keyboardEventsManager = new FocusKeyManager(radioButtons).withWrap().withHorizontalOrientation('ltr');
 
-        radioButtons.forEach((button, i) => {
-            if (this.list) {
-                button.state = this.state;
-            } else {
-                this._setProperties(button);
-            }
-
-            this._selectUnselect(button);
-
-            // finding first enabled button to set tabIndex=0
-            if (!button.disabled && !this._disabled && firstEnabledButtonIndex < 0) {
-                firstEnabledButtonIndex = i;
-            }
-
-            button.checked.pipe(takeUntil(this._destroy$)).subscribe((ev) => this._selectedValueChanged(ev));
-        });
-
-        // accessibility requirement
-        if (!this._selected && radioButtons && firstEnabledButtonIndex > -1) {
-            radioButtons.toArray()[firstEnabledButtonIndex].setTabIndex(0);
-        }
-
-        this.onChange(this.value);
+        radioButtons.changes
+            .pipe(
+                startWith(0),
```

Comment:
Isn't `startWith(radioButtons)` is more logical in this case?

Reply:
I would actually use `undefined`, but rxjs team accidentally deprecated this signature (should be fixed in next version). Changed as you asked